docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <q-page class="column q-mt-xl lp-mb--large text-white q-mx-xl">
+    <div class="row justify-center q-mb-lg">
+      <div class="input-btn">
```

Comment:
The div was the trick to  put the line rule (the bottom border) beneath the input field. I did not find any "quasar way" to do it easily. 
I could not just put that class in the input field because I needed that bottom border to be longer than the input field (by applying a padding on it). I have updated to use a "pseudo after" element instead which seems to be much more self descriptive.